superset/db_engine_specs/oracle.py

```diff
@@ -58,3 +63,16 @@ def epoch_to_dttm(cls) -> str:
     @classmethod
     def epoch_ms_to_dttm(cls) -> str:
         return "TO_DATE('1970-01-01','YYYY-MM-DD')+(1/24/60/60/1000)*{col}"
+    
```

Comment:
Please check the linter's error message. There appears to be unnecessary whitespace on this row.

Reply:
OK，Thank you

Reply:
@villebro ,There seems to be a problem afflicting the pre-commit.